lib/exercises.rb

```diff
@@ -1,19 +1,55 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n^2) where n is the number of strings? || O(n) where n is the total number of characters? 
+# Space Complexity: O(n) where a new hash is storing the n # of strings
 
 def grouped_anagrams(strings)
```

Comment:
👍  But your time complexity is O(n) since you're only looping through the words once.